File: tasks/2024/feb/14.02/removeMinimum.js
function removeSmallest(numbers) {
    let minimum = numbers.slice().sort((a, b) => a - b)[0];
    let count = 0;
    const result = [];
    for (const x of numbers) {
        if (count === 0 && minimum === x) {
            count += 1;
            continue;
        }
        result.push(x)
    }
    return result;
}

module.exports = removeSmallest;

File: tasks/2024/feb/14.02/remove_minimum.py
def remove_smallest(numbers):
    result = []
    count = 0
    sorted_list = sorted(numbers)
    print(sorted_list)
    for i in numbers:
        if i == sorted_list[0] and count == 0:
            count += 1
            continue
        result.append(i)
    return result